client/blocks/post-actions/index.jsx

```diff
@@ -99,9 +100,26 @@ PostActions.propTypes = {
 	trackTotalViewsOnClick: React.PropTypes.func,
 };
 
+const mapStateToProps = ( state, { site, post } ) => {
+	const siteId = get( site, 'ID' );
+	const isJetpack = isJetpackSite( state, siteId );
+
+	const showComments = ( ! isJetpack || isJetpackModuleActive( state, siteId, 'comments' ) ) &&
```

Comment:
No, it's fine to do separately, and after the sites-list stuff. Just noting so we don't forget. Perhaps add a comment.